bubblegum/program/src/lib.rs

```diff
@@ -785,9 +786,9 @@ fn process_collection_verification<'info>(
         &merkle_tree.key(),
         *ctx.bumps.get("authority").unwrap(),
```

Comment:
Does this now need to be "tree_authority"?

Reply:
yes good catch I knew I needed to find and replace all of those I even have a note here to do that in my text editor, but forgot to do that.

Reply:
Fixed